phoenix-tracing-webapp/README.md

```diff
@@ -3,10 +3,10 @@
  *mvn clean install*
```

Comment:
Nit: We use backticks to format command lines elsewhere, might as well switch to that here, as well.